projects/storefrontlib/src/cms-components/user/login-form/login-form.component.ts

```diff
@@ -9,16 +9,16 @@ import {
   WindowRef,
 } from '@spartacus/core';
 import { Subscription } from 'rxjs';
-import { CustomFormValidators } from '../../../shared/utils/validators/custom-form-validators';
 import { CheckoutConfigService } from '../../checkout/services/checkout-config.service';
+import { CustomFormValidators } from '../../../shared/index';
 
 @Component({
   selector: 'cx-login-form',
   templateUrl: './login-form.component.html',
 })
 export class LoginFormComponent implements OnInit, OnDestroy {
```

Comment:
Info needed about breaking changes needed:
- renamed login to submitForm

Reply:
Done